app/views/view_data/services/show.html.erb

```diff
@@ -57,14 +63,20 @@
 
           <% if @service.paper_form_url.present? %>
             <li><%= link_to 'Paper form', @service.paper_form_url %></li>
+            <!-- <li>This service is delivered by <%= link_to %></li> --> 
```

Comment:
Comment left in accidentally?

Reply:
Accidentally on purpose, it needs to link to the currently non-existent delivery organisation page, more of a reminder. Should be okay, it's next on our list of things to do.